README.md

```diff
@@ -1,17 +1,20 @@
 # OpenCL<sup>TM</sup> API C++ bindings
 
-Doxgen documentation for the cl2.hpp header is available here:
+Doxgen documentation for the bindings is available here:
 
   http://khronosgroup.github.io/OpenCL-CLHPP/
 
 Components:
 
-  * `include/CL/cl2.hpp`:
+  * `include/CL/opencl.hpp`:
     The latest, maintained, version of the C++ bindings. It should work with all
     versions of OpenCL (including 1.x). This is what most users will want.
 
+  * `include/CL/cl2.hpp`:
+    Symbolic link to `opencl.hpp` for backwards compability.

```

Comment:
Do we need to make this an actual file that just includes `opencl.hpp` rather than a symlink, so that it also works on Windows? I don't have a strong opinion about this though.

Reply:
Although, if we made it an actual file, we could also add a `#warning` message that tells developers to start using the new header, which would help with the transition.

Reply:
Don't symbolic links work on Windows these days? Anyway, I like your `#warning` idea, I'll go with that.